api/google-search/src/main/java/com/google/search/googlesearch/controller/v1/api/SearchAutoCompleteController.java

```diff
@@ -0,0 +1,35 @@
+package com.google.search.googlesearch.controller.v1.api;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.google.search.googlesearch.models.SearchAutoCompleteModel;
```

Comment:
I am not a fan of naming the package with "com.google". However, it can be discussable ofcourse.